src/components/productDetail.js

```diff
@@ -2,31 +2,44 @@ import { useDispatch } from "react-redux";
 import { formatPriceForColonCurrency } from "../service/priceService";
 import { addCartElement } from "../store"
 import "../styles/productDetail.css"
+import Star from "./star";
 
 function ProductDetail({ productInformation }) {
+
+
+
```

Comment:
Please remove those empty lines and fix the merge conflicts

Reply:
what do you mean by merge conflicts?